lib/testing/src/lib/process-services/pages/start-process.page.ts

```diff
@@ -145,9 +152,21 @@ export class StartProcessPage {
         return new FormFields();
     }
 
+    async checkSelectProcessField(): Promise<void> {
+        await BrowserVisibility.waitUntilElementIsNotVisible(this.disabledSelectProcessDropdown);
+    }
+
     async startProcess({name, processName }) {
         await this.enterProcessName(name);
         await this.selectFromProcessDropdown(processName);
         await this.clickStartProcessButton();
     }
+
+    async startProcessWithApplication({name, applicationName, processName }) {
```

Comment:
avoid use object in the firm of the method use three input parameters, we never use this kind of declaration